### ISSUE TITLE
Identify functional needs as part of normative WCAG 3.0

### ISSUE THREAD

**Issue by spanchang** [w3c/wcag3#419]
I believe a pivotal change and fundamental ingredient of WCAG 3.0 against WCAG 2.X is the reference to "functional needs". This is highlighted in the first para About WCAG 3.0 section 1.2.
It states:
"WCAG 3.0 supports a wide set of user needs, uses new approaches to testing, and allows frequent maintenance of guidelines and related content to keep pace with accelerating technology changes. WCAG 3 supports this evolution by focusing on the functional needs of users.
 These needs are then supported by guidelines that are written as outcome statements, requirements, assertions, and technology-specific methods to meet those needs".

Therefore, the conformance process should allow one to determine how the needs are met. It should be tied to the functional needs as suggested in
https://lists.w3.org/Archives/Public/w3c-wai-gl/2025OctDec/0059.html

Please consider incorporating the functional needs as part of the normative content of WCAG 3.0 in a manner similar to Functional Performance Statements (Clause 4.2) in EN 301 549.
This does not limit future versions (like WCAG 3.1 say) from adding /tweaking functional needs if circumstances warrant.
Thanks very much
Sailesh

**Comment by hidde** [comment]
Functional performance statements have now been included in the reporting section of the latest WCAG 3 draft in [4.2 Functional performance statements](https://www.w3.org/TR/wcag-3.0/#function-performance-statements).

Does this satisfy your concern?
